README.md

```diff
@@ -1,52 +1,147 @@
-## BEGIN - TO DELETE TILL END
+# reshuffle-jira-connector
```

Comment:
Do we know if the on-prem version of Jira has the same API and the cloud version? In other words, can I point this to my own Jira and it will work? We need to able to support this for Monday.

Reply:
The document says connect to any Jira instance, even locally but I haven't tried it. I'll give it a go